testsuite/integration/clustering/src/test/java/org/jboss/as/test/clustering/cluster/sso/ReplicatedSingleSignOnTestCase.java

```diff
@@ -0,0 +1,65 @@
+/*
+ * JBoss, Home of Professional Open Source.
+ * Copyright 2018, Red Hat, Inc., and individual contributors
+ * as indicated by the @author tags. See the copyright.txt file in the
+ * distribution for a full listing of individual contributors.
+ *
+ * This is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU Lesser General Public License as
+ * published by the Free Software Foundation; either version 2.1 of
+ * the License, or (at your option) any later version.
+ *
+ * This software is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public
+ * License along with this software; if not, write to the Free
+ * Software Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA
+ * 02110-1301 USA, or see the FSF site: http://www.fsf.org.
+ */
+
+package org.jboss.as.test.clustering.cluster.sso;
+
+import org.jboss.arquillian.container.test.api.Deployment;
+import org.jboss.arquillian.container.test.api.TargetsContainer;
+import org.jboss.arquillian.junit.Arquillian;
+import org.jboss.as.arquillian.api.ServerSetup;
+import org.jboss.as.test.integration.web.sso.SSOTestBase;
+import org.jboss.as.test.shared.CLIServerSetupTask;
+import org.jboss.shrinkwrap.api.Archive;
+import org.junit.runner.RunWith;
+
+/**
+ * @author Paul Ferraro
+ */
+@RunWith(Arquillian.class)
+@ServerSetup(ReplicatedSingleSignOnTestCase.ServerSetupTask.class)
+public class ReplicatedSingleSignOnTestCase extends AbstractSingleSignOnTestCase {
```

Comment:
@pferraro I can't see where this test is using the new Hot Rod session manager; is it just a test that was missing before?

Reply:
The previous ClusteredSingleSignOnTestCase was generalized so that we could have 2 variants:
* ReplicatedSingleSignOnTestCase - tests SSO using a replicated embedded cache
* RemoteSingleSignOnTestCase - tests SSO using a remote cache over hotrod

Because, the ClusteredSingleSignOnTestCase was renamed to AbstractSingleSignOnTestCase, ReplicatedSingleSignOnTestCase appears as a new file.